resources/run_adl.sh

```diff
@@ -0,0 +1,195 @@
+#!/bin/bash
+
+NAME=`basename "$0"`
+if [[ $SKIP_RUN_ADL -eq 1 ]]; then
```

Comment:
Should we document what any of these external variables--such as `SKIP_RUN_ADL`--are in the README or at least in this script?